continuum-auth/README.md

```diff
@@ -0,0 +1,19 @@
+# continuum-auth
+
+**The purpose of this service is to implement an interface to connect the eLife journal to libero reviewer**
+
+A connector service so that eLife's deployment of libero can authenticate using the existing journal login system. After successful authentication, this service will redirect to an endpoint, with a
+signed jwt token appended to the URL. This token will then be used by the client to authenticate it's subsequent HTTP requests with other services in the system. This token will contain some information
+about the user, provided by the `profiles` service.
+
+## Things to define for continuum-auth to be useful long term
+
+- We need to stabilise the interface for the auth token, so that we can swap out this service with something else
+- We need some way to manually test the login flow from end to end. This means that it needs to be:
```

Comment:
if you mean eLife infrastructure, reviewer would be deployed inside the existing `end2end` and `staging` (and `prod`) environments so there would be an actual instance of the real services to integrate against